ui/app.js

```diff
@@ -2052,6 +2052,7 @@ $(document).ready(() => {
                     <div class="associated-incidents-user-tag red-tag" data-id="${$(this).data("cid")}">Processed</div>
                     <div class="associated-incidents-user-tag red-tag" data-id="${$(this).data("cid")}">Associated</div>
                 </div>
+                <div class="modify-charges-tipp"><span class="fas fa-hand-point-right"></span> Right click below to Add/Modify Charges</div>

```

Comment:
nit: `modify-charges-tip` (one p 😜 ) or `modify-charges-label`

Otherwise nice, this at least will make it easier to know you can right click in here. 
I think there are a few other places in the mdt that have right click only behavior as well, might be good to apply this in those spots as well tomake it more obvious. Or just converting the right click behavior into dedicated buttons for the action

Reply:
Which other Places ? Only knew the modify Charges one

Reply:
Found some Right click ones

Reply:
i think there are some items that have right clicks, like for deleting items, but those probably dont need this, now that i think about it im not sure if there are other areas that need this treatment